api-report/test-runtime-utils.api.md

```diff
@@ -150,6 +150,8 @@ export class MockContainerRuntimeFactoryForReconnection extends MockContainerRun
     createContainerRuntime(dataStoreRuntime: MockFluidDataStoreRuntime, overrides?: {
         minimumSequenceNumber?: number;
     }): MockContainerRuntimeForReconnection;
+    // (undocumented)
+    getMinSeq(): number;
```

Comment:
We generally avoid abbreviations in our code. Given that the previous line is referring to the same thing, and uses the full name, its probably better to use the full name "minimumSequenceNumber". Also it would be nice for public APIs to have documentation: I know the other APIs here don't, but we should try and trend in the direction of documenting things.

Reply:
I am replacing abbreviations for local variables, but I'd avoid changing the `getMinSeq` function name, as it will cause a lot of changes in the codebase not related to this PR at all, so to keep necessary changes at minimum. Also,  changing a `minSeq` map property name of the mock class prevents building the package, probably, due to a backward compatibility requirement, not sure with this though.

Reply:
Ah, I see. This is showing up as a new undocumented member despite being overriding an existing method since its a class. I forgot classes do that in API reports unlike interfaces.